index.html

```diff
@@ -29,25 +29,112 @@
             </ul>
         </div>
     </header>
-    <main class="landing">
-        <div class="about-us">
-            <div class="social-icon">
-                <ul>
-                    <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
-                    <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
-                    <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
-                    <li><a href="#"><i class="fab fa-youtube "></i></a></li>
-                    <li><a href="#"><i class="fab fa-medium "></i></a></li>
+    <main>
+        <section class="landing">
+            <div class="about-us">
+                <div class="social-icon">
+                    <ul>
+                        <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
+                        <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
+                        <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
+                        <li><a href="#"><i class="fab fa-youtube "></i></a></li>
+                        <li><a href="#"><i class="fab fa-medium "></i></a></li>
+                    </ul>
+                </div>
+                <div class="intro ">
+                    <h1>Hey there. <br>I am Sediqullah Badakhsh</h1>
+                    <h2>I am a Software Developer</h2>
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+            </div>
+            <div>
+                <img class="landing-image " src="image/landig-image.jpg" alt="landing-image "> </div>
+        </section>
+        <section class="recent-works">
```

Comment:
- [ ] Good job so far 😄. Please consider adding all the project cards from your chosen template. Your chosen template has 7 cards but you have only 4 cards. I suggest you add the remaining cards so that your design matches that of your chosen template 💪